griptape/tasks/python_task.py

```diff
@@ -0,0 +1,15 @@
+from attr import define, field
+from griptape.artifacts import BaseArtifact, ErrorArtifact
+from griptape.tasks import BaseTextInputTask, BaseTask
+from typing import Callable
+
+
+@define
+class PythonTask(BaseTextInputTask):
+    run_fn: Callable[[BaseTask], BaseArtifact] = field(kw_only=True)
```

Comment:
I think we can reduce the typing scope from `BaseTask` to `PythonTask` since it's always passing itself to `run_fn`